src/app/(legal-documents)/provider-terms-of-use/page.js

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import { PolicyContainer } from "@/app/(legal-documents)/parent-terms-of-use/page";
+import {
+    BetweenSection,
+    BackgroundSection,
+    DefinitionSection,
+    ProviderObligationSection,
+    ServicesAndPlatformSection,
+    IntellectualPropertySection,
+    PaymentSection,
+    DataProtectionSection,
+    ConfidentialitySection,
+    TermAndTerminationSection,
+    WarrantiesAndLiabilitySection,
+    GeneralSection
+} from "@/app/(legal-documents)/provider-service-agreement/page";
+
+export default function TermsAndConditionsPage() {
```

Comment:
Why do we have two words: terms of use and terms and condititons? Is it the same? Than should be named identically.